about.html

```diff
@@ -0,0 +1,218 @@
+          </p>
+        </div>
+        <div class="row about-heading-3">
+          <h3>
+            If you have any question about the conference, please feel free to contact us
+           
+          </h3>
+        </div>
+        <div class="row about-heading-4">
+          <h4>info@afrivandanceconference.com</h4>
+        </div>
+      </div>
+    </section>
+
+     <!-- Logo detail section start from here -->
+     <div id="logo-detail-section" class="contianer-fluid logo-section">
+      <div class="container">
+        <div class="row">
+          <div class="col-12 d-flex justify-content-center logo-text">
+            Creative Commons 
+            Global Summit 2015 Logo
+          </div>
+        </div>
+        <div class="row d-flex justify-content-center">
+          <div class="col-1 logo-text-line">line</div>
+        </div>
+        <div class="row logo-para-heading">
+          <p>
+            The logo of CC Global Summit 2015 was decided through the logo competition from 8, June to 7, July.
+After online public voting and Selection Committee’s judging process, the logo from Naresh Agrawal won the prize.
+After consultation with Mr. Agrawal, we optimized the prize-winning-logo for this year’s summit as follows:
+          </p>
+        </div>
+        <div class="row webly-logo">
+          <img src="images/creative_middle.png" alt="" />
+        </div>
+      </div>
+    </div>
+    <!-- Past Batch section start from here -->
+    <section id="past-Batch-section" class="contianer-fluid past-section">
+      <div class="container">
+        <div class="row">
+          <div class="col-12 d-flex justify-content-center past-text">
+            See the Past conferences
+          </div>
+        </div>
+        <div class="row d-flex justify-content-center">
+          <div class="col-1 past-text-line">line</div>
+        </div>
+        <div class="row past-para-heading">
+          <p>
+            Take a look of the past conferences which took place in 2021 and 2022.
+          </p>
+        </div>
+        <div class="past-pic">
+          <div class="pastimage1">
+            <h2>2021</h2>
+            <p>Award Cermony in California</p>
+          </div>
+          <div class="pastimage2">
+            <h2>2022</h2>
+            <p>Award Cermony in America</p>
+          </div>
+        </div>
+      </div>
+    </section>
+
+     <!-- Footer section start from here -->
+
+    <footer id="footer" class="continer-fluid footer-section2">
+      <div class="contianer footer-container ">        
+        <p><img src="images/creative.png" width="80%"  alt=""/></p><p>2015 Creative Commons Korea.Some Rights Reserved. 달리 정하지 않은 웹사이트의 모든 내용은 크리에이티브 커먼즈 저작자표시 2.0 대한민국 라이선스에 따라 이용할 수 있습니다.
+          자세한 이용기능 내용과 허락조건은 http://creativecommons.org/licences/by/2.0/kr 를 보세요. </p>
+      </div>
+    </footer>
```

Comment:
- [ ]  In the process of removing the partner section, I observed that a little bug was introduced. 👇 Please kindly fix it and ensure to design the footer element nicely. 👍 

<img width="960" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/235294519-31100f1f-634e-478a-8fbd-3b0776820c09.png">